docs/reactivity/reactive-components.md

```diff
@@ -0,0 +1,103 @@
+# Build reactive components with Oryx
+
+## Integrate with domain logic
+
+Components can use domain logic to fetch data. Domain services provide a convenient API to request data, e.g. in case of loading product data, the Product Service can be leveraged:
```

Comment:
But I'd say it's our strong recommendation to use domain controllers to fetch domain data.

Reply:
for sure, but we're building the story here.

Reply:
so I keep it as is